app/models/user.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const user = sequelize.define(
```

Comment:
name of the variable where you define the model must start with a capital letter, the same in the define